addons/sale/tests/test_payment_flow.py

```diff
@@ -24,6 +24,10 @@ def setUpClass(cls):
         cls.currency = cls.sale_order.currency_id
         cls.partner = cls.sale_order.partner_invoice_id
 
+        cls.provider.journal_id.inbound_payment_method_line_ids.filtered(lambda l:
+            l.payment_provider_id == cls.provider
+        ).payment_account_id = cls.inbound_payment_method_line.payment_account_id
```

Comment:
Isn't it boring to be forced to do that all the time in every test?
It also means that it won't be made automatically for the users anymore.

Reply:
> Isn't it boring to be forced to do that all the time in every test?

Yes, but the point will be to not do it anymore in the future. For now I was mainly concentrating on not breaking anything.
In the future, we will be smarter on which methods require an account or not, and maybe create it automatically when needed